Kajabi/sage-lib
# Input Group added rails component

## Description


## Test notes

No visual diff

### Steps for testing
1. Visit input group rails page: http://localhost:4000/pages/object/input_group
2. Verify demo and docs
